packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx

```diff
@@ -3,8 +3,48 @@ id: reaction
 title: Reaction
 ---
 
-:::warning
+Reaction component is used to display emojis in real-time for a specified amount of time. You can utilise this functionality to notify other participants that you want to speak by "raising hand". Our default sorting algorithm will push participants with raised hand to the top of the list for better visibility. Learn more about reaction events and their customization in the [Reactions & Custom Events guide](../../../guides/reactions-and-custom-events) and see how sorting works in the [Participant sorting guide](../../../guides/sorting-api).
```

Comment:
I'd list the supported reactions here, reading that paragraph made me think that it's for raising hand only.

Reply:
Good point, will add, thanks! 👍